### PR TITLE
fix: apply reversed sorting in a column

### PR DESCRIPTION
**Pull request by smyrn96**
## Description

Default sorting order for table columns

## Changes

Introduce a new prop `sortOrder` in Colunm type in order to specify the default sorting order in columns

### CODE REVIEW

File: src/components/Table/components/Header.tsx [hunk 1]
```diff
@@ -20,14 +20,15 @@ const Header: FC<ChildrenProps> = ({ selectable = false, state, dispatch, onSort
...
     if (sorting) {
       // new sorting object
       const newSorting: Sorting = {
         column: accesor,
-        // when select new column to sort apply ascending order
+        // when select new column to sort apply ascending order or descending if reversedSorting is enabled
         // when select the same column change the current order
-        isDescending: sorting.column !== accesor ? false : !sorting.isDescending,
+        isDescending:
+          sorting.column !== accesor ? (reversedSorting ? true : false) : !sorting.isDescending,
```
Comment (xanderantoniadis) [inline src/components/Table/components/Header.tsx:31]: Try this instead:
isDescending: sorting.column !== accesor ? reversedSorting : !sorting.isDescending
reversedSorting should contain true or false. 

Check if it works correctly.

Comment (smyrn96) [reply src/components/Table/components/Header.tsx]: Fixed: [986d480](https://github.com/epignosis/gnosis/pull/172/commits/986d48039d7ebf1279b94918318ae9369f0830aa)

File: src/components/Table/types.ts [hunk 2]
```diff
@@ -9,8 +9,11 @@ export type Column = {
   classNames?: string[];
   sortableHeader?: boolean;
   maxWidth?: number;
+  sortOrder: sortOrderType;
```
Comment (ThrasyvoulosKafasis) [inline src/components/Table/types.ts:12]: Make sortOrder omittable

File: src/components/Table/types.ts [hunk 3]
```diff
@@ -9,8 +9,11 @@ export type Column = {
...
 };
 
+export type sortOrderType = "asc" | "desc";
```
Comment (ThrasyvoulosKafasis) [inline src/components/Table/types.ts:15]: No need to use additional type. Also sortOrderType is not a very goo name.

File: src/components/Table/components/Header.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React, { FC } from "react";
 import Checkbox from "../../FormElements/CheckboxGroup/Checkbox";
 import { IconChevronDownSVG, IconChevronUpSVG } from "../../../icons/index";
-import { Sorting } from "../types";
+import { Sorting, sortOrderType } from "../types";
```
Comment (ThrasyvoulosKafasis) [inline src/components/Table/components/Header.tsx:4]: You can get the sort order type from Column type.

**Review by ThrasyvoulosKafasis — CHANGES_REQUESTED** [review]

File: src/components/Table/types.ts (hunk 2 above)
Comment (smyrn96) [reply src/components/Table/types.ts]: Fixed: [b472de7](https://github.com/epignosis/gnosis/pull/172/commits/b472de72685d7feb48534946d64ccba0afb4b2e9)

File: src/components/Table/types.ts (hunk 3 above)
Comment (smyrn96) [reply src/components/Table/types.ts]: Fixed: [b472de7](https://github.com/epignosis/gnosis/pull/172/commits/b472de72685d7feb48534946d64ccba0afb4b2e9)

File: src/components/Table/components/Header.tsx (hunk 4 above)
Comment (smyrn96) [reply src/components/Table/components/Header.tsx]: Fixed: [b472de7](https://github.com/epignosis/gnosis/pull/172/commits/b472de72685d7feb48534946d64ccba0afb4b2e9)

**Review by ThrasyvoulosKafasis — APPROVED** [review]

**Pull request merged**